pkg/aws/eni/routing/routing.go

```diff
@@ -122,6 +129,73 @@ func Install(ip net.IP, info *RoutingInfo, mtu int, masq bool) error {
 	return nil
 }
 
+// Delete removes the ingress and egress rules that control traffic for
+// endpoints. Note that the routes within these rules are not deleted as they
+// can be reused when another endpoint is created on the same node. The reason
+// for this is that each ENI device has an index called ifindex which is used
+// as the table ID when these rules are created. The routes are created inside
```

Comment:
fwiw the `ifindex` here is the `interface index`, ie "network interface" / "network device" which probably helps put the rest of this description in context :)

Reply:
Right, I'll clarify the wording as it currently sounds like "ifindex" is an ENI-specific thing.